### PR TITLE
Fix/kbqa

### PR DESCRIPTION
**Pull request by dmitrijeuseew**
resolving  #883

### CODE REVIEW

File: deeppavlov/configs/ner/ner_kb_rus.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "dataset_reader": {
     "class_name": "sq_reader",
-    "data_path": "/home/dmitry/download/ner_ontonotes"
+    "data_path": "{DOWNLOADS_PATH}/ner_sq/SQ_rus_dataset_zs.pckl"
```
Comment (yoptar) [inline deeppavlov/configs/ner/ner_kb_rus.json]: Are you sure that you don't need to change any code in the `sq_reader` after changing `data_path` format?

Comment (StrikerRUS) [reply deeppavlov/configs/ner/ner_kb_rus.json]: Where `sq_reader` can be found? GitHub search gives nothing: https://github.com/deepmipt/DeepPavlov/search?q=sq_reader+path:/deeppavlov/dataset_readers&unscoped_q=sq_reader+path:/deeppavlov/dataset_readers

Comment (yoptar) [reply deeppavlov/configs/ner/ner_kb_rus.json]: Thank you, @StrikerRUS for noticing! It's in this pull request now.

Comment (StrikerRUS) [reply deeppavlov/configs/ner/ner_kb_rus.json]: Thanks for fixing this!

**Pull request merged**